lua/nvim-treesitter/ts_utils.lua

```diff
@@ -156,6 +156,21 @@ function M.get_root_for_position(line, col, root_lang_tree)
   return nil, nil, lang_tree
 end
 
+-- Filters node list based on intersection with row and column
+--
+-- @param nodes the list of treesitter nodes
+-- @param row intersecting y column
+-- @param col intersecting x column
```

Comment:
I think we use three dashes for doc comments here :
```suggestion
--- Filters node list based on intersection with row and column
---
--- @param nodes the list of treesitter nodes
--- @param row intersecting y column
--- @param col intersecting x column
```